### PR TITLE
Modifications in LearnBlock.py and tooltips in .conf files

### PR DESCRIPTION
**Pull request by amrmoastafa**
Corrected misspelled functions in 'LearnBlock.py' , added all the missing English language tooltip to all the '.conf' filed in the 'blocksConfig' folder

### CODE REVIEW

File: learnbot_dsl/blocksConfig/default/Motor.conf [hunk 1]
```diff
@@ -112,7 +112,7 @@
         },
         "tooltip": {
             "ES": "Establece la orientaci\u00f3n actual a 0\u00ba",
```
Comment (pilarbachiller) [inline learnbot_dsl/blocksConfig/default/Motor.conf]:
```suggestion
            "ES": "Establece la orientación actual a 0",
```

File: learnbot_dsl/blocksConfig/default/Motor.conf [hunk 2]
```diff
@@ -112,7 +112,7 @@
...
-            "EN": ""
+            "EN": "Set the robot orientation to a 0\u00ba."
```
Comment (pilarbachiller) [inline learnbot_dsl/blocksConfig/default/Motor.conf]:
```suggestion
            "EN": "Set the robot orientation to 0."
```

File: learnbot_dsl/blocksConfig/default/Motor.conf [hunk 3]
```diff
@@ -139,7 +139,7 @@
         },
         "tooltip": {
             "ES": "Sit\u00faa la base del robot en una cierta orientaci\u00f3n",
```
Comment (pilarbachiller) [inline learnbot_dsl/blocksConfig/default/Motor.conf]:
```suggestion
            "ES": "Sitúa la base del robot en una cierta orientación",
```

**Review by pilarbachiller — CHANGES_REQUESTED** [review]
Thank you for your contribution. Please, review my suggestions.

File: learnbot_dsl/blocksConfig/default/Motor.conf (hunk 3 above)
Comment (amrmoastafa) [reply learnbot_dsl/blocksConfig/default/Motor.conf]: thank you , I edited them as requested

**Pull request merged**